pacstall

```diff
@@ -144,10 +144,18 @@ function ask() {
     fi
 
     # Check if the reply is valid
+    
```

Comment:
It's missing a check for `no default`
`if ask "question"` enters an infinite loop